packages/manager/modules/pci/src/projects/new/index.html

```diff
@@ -0,0 +1,9 @@
+<div class="pci-project-new h-100">
+    <div class="container h-100">
+        <div class="row minh-100">
```

Comment:
```suggestion
        <div class="row min h-100">
```

Also, all those `h-100` are needed ?